PERMISSIONS.md

```diff
@@ -0,0 +1,60 @@
+# Permissions
+
+## Repositories
+
+| Repository                               | Everyone | @nodejs/web | @nodejs/nodejs-website | @nodejs/web-infra | @nodejs/web-admins | Notes                          |
+| ---------------------------------------- | -------- | ----------- | ---------------------- | ----------------- | ------------------ | ------------------------------ |
+| **[nodejs/discord-status-worker][]**     | Read     | -           | -                      | Maintain          | Admin              |                                |
+| **[nodejs/doc-kit][]**                   | Read     | -           | Write                  | Maintain          | Admin              |                                |
+| **[nodejs/node.js.org][]**               | Read     | Write       | -                      | Maintain          | Admin              |                                |
+| **[nodejs/nodejs.dev][]**                | Read     | -           | Write                  | Maintain          | Admin              | This repository is _archived_. |
+| **[nodejs/nodejs.org][]**                | Read     | -           | Write                  | Maintain          | Admin              |                                |
+| **[nodejs/nodejs.org-archive][]**        | Read     | Write       | -                      | Maintain          | Admin              | This repository is _archived_. |
+| **[nodejs/release-cloudflare-worker][]** | Read     | -           | -                      | Maintain          | Admin              |                                |
+| **[nodejs/web-team][]**                  | Read     | Triage      | Maintain               | -                 | Admin              |                                |
```

Comment:
I don't think website team should have maintain perms here.

Reply:
I'm not documenting what _should_ exist. I'm documenting what is currently set up. This can be amended if those permissions change